src/DataFrames.jl

```diff
@@ -74,6 +74,10 @@ if VERSION < v"1.2"
     export hasproperty
 end
 
+if VERSION < v"1.1"
+    import Future.copy!
```

Comment:
I think this should be enough since you don't overload the function:
```suggestion
    using Future: copy!
```

Reply:
This will also require a dependency on Future declared in Project.toml deps.

Reply:
Actually https://github.com/JuliaData/DataFrames.jl/pull/1974 just did that, so you can just drop this block AFAICT.

Reply:
and you need to rebase this PR then.